Clockify.Net/Clockify.Net.csproj

```diff
@@ -14,6 +13,7 @@
     <PackageVersion>2.2.0</PackageVersion>
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
Needed for Math.Clamp()

Reply:
This got reset again to `netstandard2.0` and a custom `Clamp` method was instead created, because we use the package in .NET Framework.